docs/stories/04-components/LinkButton.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import { LinkButton } from '@strapi/design-system';
+
+import * as LinkButtonStories from './LinkButton.stories';
+
+<Meta of={LinkButtonStories} />
+
+# LinkButton
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Usage with other routing libraries](#usage-with-other-routing-libraries)
+- [Props](#props)
+
+## Overview
+
+A useful wrapper around the [`Button`](../?path=/docs/components-button--docs) component that allows users to navigate
+to a different location. By default, the `tag` prop is the [`BaseLink`](../?path=/docs/primitives-baselink--docs)
+component, but can be changed to integrate with routing libraries.
+
+[View source](https://github.com/strapi/design-system/tree/main/packages/design-system/src/LinkButton)
+
+## Usage
+
+```js
+import { LinkButton } from '@strapi/design-system';
+```
+
+<Canvas of={LinkButtonStories.Base} />
+
+## Usage with other routing libraries
+
+### React Router
+
+To use the LinkButton component with a routing library (e.g. react-router-dom), you'll need to pass the `NavLink`
+component to the `tag` prop in order to replace the default HTML anchor `<a />`. You'll now be able to pass all
+`NavLink` props.
```

Comment:
Are the types smart enough now to understand the props of the tags are expected? I believe that wasn't the case in v1

Reply:
It does indeed 🪄